bricksrc/equipment.py

```diff
@@ -448,21 +468,35 @@
                     TAG.Heat,
                     TAG.Equipment,
                 ],
-                "parents": [BRICK.Hot_Water_System, BRICK.Water_Valve],
+                "parents": [BRICK.Water_Valve],
             },
         },
     },
     "Cooling_Valve": {"tags": [TAG.Valve, TAG.Cool, TAG.Equipment]},
     "Water_Valve": {
         "tags": [TAG.Valve, TAG.Water, TAG.Equipment],
+        SKOS.definition: Literal(
+            "Building water is provided by utility company and sourced through a master building feed."
```

Comment:
This seems more appropriate for a definition of the substance itself rather than the valve; the valve mediates the flow of water